ocs_ci/helpers/dr_helpers.py

```diff
@@ -1288,7 +1288,7 @@ def restore_backup():
     config.switch_ctx(restore_index)
 
 
-@retry(UnexpectedBehaviour, tries=40, delay=5, backoff=5)
+@retry(UnexpectedBehaviour, tries=25, delay=5, backoff=5)
```

Comment:
same as above, lets reduce backoff to 1 in this case? because 25 tries with delay 5 and even with backoff = 2 it will last 2+ years

Reply:
@DanielOsypenko it won't, as I limited maximum delay time to be 10 minutes , so once it should go over 10 minutes it's limited to max 10 minutes.  Based on pre-commit hook check , this will new values will be under 4 hours max.